dispatcher.py

```diff
@@ -388,7 +389,7 @@ def _run_worker(self, worker_id, tcp_port_range):
         os.environ['TEST_RUN_TCP_PORT_END'] = str(tcp_port_range[1])
         color_stdout.queue = self.result_queue
         worker = self.gen_worker(worker_id)
-        worker.run_all(self.task_queue, self.result_queue)
+        worker.run_all(self.task_queue, self.result_queue, self.is_fragile)
```

Comment:
Several points (not critical, more kind of stylistic thoughts):

1. It seems more natural to pass it to gen_worker() function (because it is property of a worker in fact). If you'll do, don't forget about 'get _id as an arg' comment in worker.py.
2. For worker it is more like 'whether task retries are allowed' property, because it is quite abstract from tests. Maybe we can name it better: say, `enable_retries`.
3. However we can just lean on `retries` suite.ini value, it is available in `run_loop()`. There are two cons. First, we don't need to pass the flag over several calls deep into `run_loop()`. Second, it is quite natural to just follow `retries`.

Reply:
Right, this flag is not needed and I completely removed it.